test cases/nasm/3 nasm only/meson.build

```diff
@@ -0,0 +1,17 @@
+project('nasm only')
+
+if not add_languages('nasm', required: false)
+  error('MESON_SKIP_TEST: nasm not found')
+endif
+
+if meson.backend().startswith('vs')
+  error('MESON_SKIP_TEST: VS backend does not recognise NASM yet')
+endif
+
+sources = files('dummy.asm')
+
+dummy = library(
+    'dummy',
+    sources,
+    vs_module_defs: 'dummy.def',
+)
```

Comment:
I'm not sure this unit test is needed, isn't this already covered with "2 asm language" and "4 nasm and c together"?

Reply:
"2 asm language" is Linux only: https://github.com/mesonbuild/meson/pull/11127#discussion_r1044447712

The situation re: "4 nasm and c together" was explained in https://github.com/mesonbuild/meson/pull/11127#discussion_r1040931818